src/components/EventTheme/index.tsx

```diff
@@ -1,14 +1,23 @@
 import clsx from "clsx";
 
+import colors from "./colors";
 import styles from "./index.module.css";
 
 export interface EventThemeProps {
   children: React.ReactNode;
   event: string;
 }
 
+export interface EventColorProps {
+  event: string;
+}
+
 export function EventTheme({ children, event }: EventThemeProps) {
   return (
     <div className={clsx(styles.eventTheme, styles[event])}>{children}</div>
   );
 }
+
+export function EventColors({ event }: EventColorProps) {
+  return colors[event];
+}
```

Comment:
[Suggestion] This component and its prop type are named & structured like a React function component, but it's not actually used like one. It's used more like a _hook_. I think structuring it like one would be more idiomatic (and less code):

```suggestion
export function useEventColors(event: string) {
  return colors[event];
}
```

Thoughts?